web-common/src/features/dashboards/filters/Filters.svelte

```diff
@@ -405,6 +409,10 @@
                 onToggleFilterMode={() => toggleDimensionFilterMode(name)}
                 onSelect={(value) =>
                   toggleDimensionValueSelection(name, value, true)}
+                onBulkSelect={(values) =>
+                  applyDimensionBulkSearch(name, values)}
```

Comment:
We need to use consistent terminology. It's hard to keep track of "bulk select" vs "bulk search" vs  "in list".